manticore/ethereum/__init__.py

```diff
@@ -721,12 +739,20 @@ def _get_uniq_name(self, stem):
         assert name not in self.accounts
         return name
 
+    def _all_addresses(self):
+        """ Returns all addresses in all running states """
+        ret = set()
+        for state in self.running_states:
+            ret |= set(state.platform.accounts)
+        return ret
```

Comment:
Heh
```
In [9]: set().union([1,2,3],[4,5,6])
Out[9]: {1, 2, 3, 4, 5, 6}

In [10]: set().union(*(i for i in [[1,2,3],[4,5,6]]))
Out[10]: {1, 2, 3, 4, 5, 6}
```

It could be written as `return set().union(*(state.platform.accounts for state in self.running_states))`